src/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js

```diff
@@ -25,10 +26,10 @@ const useStyles = makeStyles(theme => ({
 const DashboardPlaceholder = (props) => {
   const classes = useStyles();
   const { t } = useTranslation();
-  const { labelKey, defaultLabel, icon } = props;
+  const { labelKey, defaultLabel, icon, button, downloadCumulatedLogs } = props;
```

Comment:
The "button" property should be removed here too (not used, and not declared in the propTypes)

Reply:
Done